src/NServiceBus.Metrics/Reporters/NServiceBusMetricReport.cs

```diff
@@ -16,13 +16,13 @@
 
 class NServiceBusMetricReport : MetricsReport
 {
-    string destination;
+    UnicastAddressTag destination;
     IDispatchMessages dispatcher;
 
     public NServiceBusMetricReport(IDispatchMessages dispatcher, string destination)
     {
         this.dispatcher = dispatcher;
-        this.destination = destination;
+        this.destination = new UnicastAddressTag(destination);
```

Comment:
Should we be requiring the user to pass in a `UnicastTag`? Do we do that anywhere else?